components/forms/Label/Label.tsx

```diff
@@ -30,13 +30,16 @@ export const Label = (props: LabelProps): React.ReactElement => {
 
   const childrenElements = (
     <>
-      {children}{" "}
-      {required ? (
-        <span data-testid="asterisk" aria-hidden>
-          *
-        </span>
-      ) : null}
-      {required ? <i className="visually-hidden">{t("required-field")}</i> : null}
+      {children}
+      {required && (
+        <>
```

Comment:
code style. Rather then use a react fragment within a react fragment . It's best to split out as it was before